### PR TITLE
Feature / Add changesets

### PR DESCRIPTION
**Pull request by Santiago-Souto**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,5 +40,8 @@
     "lerna": "^5.5.0",
     "lint-staged": "^10.5.4",
     "npm": "^8.11.0"
+  },
+  "dependencies": {
```
Comment (vincentsong) [inline package.json:44]:
```suggestion
  "devDependencies": {
```

File: .changeset/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Changesets
+
+Hello and welcome! This folder has been automatically generated by `@changesets/cli`, a build tool that works
```
Comment (vincentsong) [inline .changeset/README.md:3]: Please update it to a meaningful log

File: .changeset/red-flowers-wait.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@millicast/sdk": patch
+---
+
+Added changesets for changelog control
```
[the remark this answers is not in the document]
Comment (Santiago-Souto) [reply .changeset/red-flowers-wait.md:5]: As #306 is already merged, I will add it here.

[the remark this answers is not in the document]
Comment (vincentsong) [reply .changeset/red-flowers-wait.md:5]: @Santiago-Souto  no worries, I will check out that branch and add it

**Pull request merged**